pkg/installer/generateconfig.go

```diff
@@ -150,115 +150,117 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 	}
 
 	installConfig := &installconfig.InstallConfig{
-		Config: &types.InstallConfig{
-			TypeMeta: metav1.TypeMeta{
-				APIVersion: "v1",
-			},
-			ObjectMeta: metav1.ObjectMeta{
-				Name: domain[:strings.IndexByte(domain, '.')],
-			},
-			SSHKey:     sshkey.Type() + " " + base64.StdEncoding.EncodeToString(sshkey.Marshal()),
-			BaseDomain: domain[strings.IndexByte(domain, '.')+1:],
-			Networking: &types.Networking{
-				MachineNetwork: []types.MachineNetworkEntry{
-					{
-						CIDR: *ipnet.MustParseCIDR("127.0.0.0/8"), // dummy
-					},
-				},
-				NetworkType: SoftwareDefinedNetwork,
-				ClusterNetwork: []types.ClusterNetworkEntry{
-					{
-						CIDR:       *ipnet.MustParseCIDR(m.oc.Properties.NetworkProfile.PodCIDR),
-						HostPrefix: 23,
-					},
+		AssetBase: installconfig.AssetBase{
+			Config: &types.InstallConfig{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: "v1",
 				},
-				ServiceNetwork: []ipnet.IPNet{
-					*ipnet.MustParseCIDR(m.oc.Properties.NetworkProfile.ServiceCIDR),
+				ObjectMeta: metav1.ObjectMeta{
+					Name: domain[:strings.IndexByte(domain, '.')],
 				},
-			},
-			ControlPlane: &types.MachinePool{
-				Name:     "master",
-				Replicas: to.Int64Ptr(3),
-				Platform: types.MachinePoolPlatform{
-					Azure: &azuretypes.MachinePool{
-						Zones:            masterZones,
-						InstanceType:     string(m.oc.Properties.MasterProfile.VMSize),
-						EncryptionAtHost: m.oc.Properties.MasterProfile.EncryptionAtHost == api.EncryptionAtHostEnabled,
-						VMNetworkingType: masterVMNetworkingType,
-						OSDisk: azuretypes.OSDisk{
-							DiskEncryptionSet: masterDiskEncryptionSet,
-							DiskSizeGB:        1024,
+				SSHKey:     sshkey.Type() + " " + base64.StdEncoding.EncodeToString(sshkey.Marshal()),
+				BaseDomain: domain[strings.IndexByte(domain, '.')+1:],
+				Networking: &types.Networking{
+					MachineNetwork: []types.MachineNetworkEntry{
+						{
+							CIDR: *ipnet.MustParseCIDR("127.0.0.0/8"), // dummy
+						},
+					},
+					NetworkType: SoftwareDefinedNetwork,
```

Comment:
Can we make this lowercase?  Otherwise the value is exported from the package and I don't think that's what we should be doing for future stuff.

Reply:
`SoftwareDefinedNetwork`

Reply:
we can, it was as it previously, but yeah we can definitely make it lowercase now.